src/components/layouts/MainLayout.jsx

```diff
@@ -75,16 +80,37 @@ export const MainLayout = () => {
                 </OpacityButton>
               </BurgerWrap>
               <DesktopTitleWrap>
-                <Text
-                  type='h1'
-                  fontSize={32}
-                  fontWeight={700}
-                  lineHeight={1}
-                  color={colors.text}
-                  style={{ textShadow: shadows.headingShadow }}
-                >
-                  {routeTitles[selectedRoute]}
-                </Text>
+                {width >= 1024 && !!value && value === 'day' && (
```

Comment:
better to move this condition to variable

const showTitleGooseLogo = width >= 1024 && window.location.pathname.includes(ROUTES.CALENDAR) && params.get(CALENDAR_TYPE_QUERY) === 'day'